backend/app.mjs

```diff
@@ -10,8 +12,46 @@ app.use(express.urlencoded({ extended: true }));
 app.use(helmet());
 app.use(cors());
 
+const myBlockchain = new Blockchain();
+
+const block1 = new Block(1, '02/01/2024', { amount: 4 });
+myBlockchain.addBlock(block1);
+
+const block2 = new Block(2, '03/01/2024', { amount: 8 });
+myBlockchain.addBlock(block2);
+
+console.log('Blockchain valid?', myBlockchain.isChainValid());
+console.log(JSON.stringify(myBlockchain, null, 2));
+
+app.get('/', (req, res) => {
```

Comment:
Set up a router file for these, instead of having them in the app.mjs file.
- Move the code from line 26 to 52 to a new file
- The new file should be named "blockchain-routes.mjs" or similar
- The new file should be in the "routers" folder
- Create and export a Express router; example code:
```
import express from 'express';

const router = express.Router();

router.route('/').get(importedControllerFunctionGoesHere);

export default router;
```
- import the router in the app.mjs file and set up a `app.use()` to handle the route; example code:
```
import blockchainRouter from './routers/blockchain-routes.mjs';

app.use('/api/v1/blockchain', blockchainRouter);
```